CMakeLists.txt

```diff
@@ -12,6 +12,7 @@ find_package(dataformats REQUIRED)
 find_package(dfmessages REQUIRED)
 find_package(folly REQUIRED)
 find_package(TBB REQUIRED)
+find_package(Boost 1.70.0 COMPONENTS unit_test_framework REQUIRED)
```

Comment:
are you sure we need this? I don't see this explicit call in other packages... e.g.: appfwk package.

Reply:
Hi Roland! Thanks for the review. I am actually not sure this line is needed, but I have seen it being used in the dataformats packages. You are right that perhaps we can build without it but I haven't checked this.

The main changes in the PR are related to add logging levels so that the debug printouts can be tuned on and off. The RawWIBTp unit test is not finished yet. I am going to fill it in with the complete set of tests but for now I've just added the 
directory and the file and checked that it builds.